db/dcrsqlite/apisource.go

```diff
@@ -973,8 +972,8 @@ func (db *wiredDB) GetExplorerBlock(hash string) *explorer.BlockInfo {
 	tickets := make([]*explorer.TxBasic, 0, block.FreshStake)
 
 	for _, tx := range data.RawSTx {
-		msgTx := txhelpers.MsgTxFromHex(tx.Hex)
-		if msgTx == nil {
+		msgTx, err := txhelpers.MsgTxFromHex(tx.Hex)
+		if err != nil {
```

Comment:
please include `err` in the log message below since we're not returning it from this functions